its/ruling/src/test/resources/expected/python-S930.json

```diff
@@ -22,9 +26,6 @@
 'project:pecos/examples/qp2q/utils/analyse_eval_results.py':[
 241,
 ],
-'project:tornado-2.3/tornado/escape.py':[
-137,
```

Comment:
This one was an FP but we are now detecting the correct number of expected arguments